src/envo/envo-edit.owl

```diff
@@ -3749,6 +3801,7 @@ Declaration(AnnotationProperty(<http://www.geneontology.org/formats/oboInOwl#cre
 Declaration(AnnotationProperty(<http://www.geneontology.org/formats/oboInOwl#creation_date>))
 Declaration(AnnotationProperty(<http://www.geneontology.org/formats/oboInOwl#date>))
 Declaration(AnnotationProperty(<http://www.geneontology.org/formats/oboInOwl#default-namespace>))
+Declaration(AnnotationProperty(<http://www.geneontology.org/formats/oboInOwl#exactSynonym>))
```

Comment:
Same here - does this not already exist?

Reply:
That was broken. fixing now

Reply:
(should have been http://www.geneontology.org/formats/oboInOwl#hasExactSynonym)

Reply:
I'm not sure if this is the issue, but like I had previously said to @laurenechan in your template if you could please use the APs as listed in the [example sheet](https://docs.google.com/spreadsheets/d/1K5GWxpSF2s397FMjdb9nGicC8xl10Xd1-M3-4fOnCts/edit#gid=0) from the [ENVO-Robot-template-and-merge-workflow](https://github.com/EnvironmentOntology/envo/wiki/ENVO-Robot-template-and-merge-workflow)
e.g. we have `AL oboInOwl:hasExactSynonym@en`

Reply:
Yes that was the issue.. My fault, I was not aware of the ENVO template and gave lauren a broken template to start from. Fixed now in the edit version. Got rid of the templates.